app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, list_of_cars: list) -> float:
+        earnings = 0
+        for car in list_of_cars:
+            if car.clean_mark < self.clean_power:
+                earnings += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+        return earnings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        washing_difference = self.clean_power - car.clean_mark
+        fraction = self.average_rating / self.distance_from_city_center
+        price_for_wash_car = round(
+            car.comfort_class
+            * washing_difference * fraction, 1)
+        return price_for_wash_car
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark <= self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def rate_service(self, new_rating: int) -> None:
+        total_rating = self.average_rating * self.count_of_ratings
+        overall_rating = total_rating + new_rating
+        self. average_rating = round(
+            overall_rating / (self.count_of_ratings + 1), 1)
```

Comment:
Fix indentation here

```suggestion
        self. average_rating = round(
            overall_rating / (self.count_of_ratings + 1), 1
            )
```